project/budget.py

```diff
@@ -0,0 +1,32 @@
+from project import db
+from project.models import Budget
+
+
+def fetch_budget(id):
+    return db.session.get(Budget, id)
+
+
+def fetch_budget_id(user_id):
+    budget_id = (
+        db.session.query(Budget).filter_by(user_id=user_id).first()
+    )
+    return budget_id.id if budget_id else budget_id
+
+
+def set_budget(user_id):
+    new_budget = Budget(amount=0.00, user_id=user_id)
+    db.session.add(new_budget)
+    db.session.commit()
+    return new_budget
+
+
+def add_budget(added_amount, user_id):
+    budget = fetch_budget(user_id)
```

Comment:
Rename function variables as fetch_budget takes in the budget_id(or id in this case), not the user_id.